examples/README.md

```diff
@@ -0,0 +1,3 @@
+# Examples
+
+This folder contains examples on how to use `libf3d`. All the examples are using `find_package(f3d)` so they are intended to be built from an existing f3d installation.
```

Comment:
this file may need to be excluded from the jekyll doc, it mnay be grabbed autmatically.

Reply:
lets merge and check then